src/ol_concourse/pipelines/open_edx/mfe/pipeline.py

```diff
@@ -154,6 +154,16 @@ def mfe_job(
         )
     )
 
+    mfe_smoot_design_overrides = (
+        """
+    npm pack @mitodl/smoot-design
+    tar -xvzf mitodl-smoot-design*.tgz
+    mv package mitodl-smoot-design
```

Comment:
@ChristopherChudzicki Does this look good to you? I have removed the version to keep it generic for future versions.

Reply:
I sort of think we should keep the version, or at least do
```
npm pack @mitodl/smoot-design^3.4
tar -xvzf mitodl-smoot-design*.tgz
mv package mitodl-smoot-design
```
If we introduce a breaking change, it can be v4.

-----------------

I do think that this is the wrong place to keep the version. I was talking to @blarghmatey about ways we could handle this, e.g., we could put a `env-config-jsx-setup.sh` script in `ol-infrastracture` aside `env.config.jsx`, and specify the commands (including version) there. Or if not a shell script, just store the version we want.

Reply:
_But I think putting it here, with a version---either `^3.4` or `3.4`---is fine for now.